OpenRA.Mods.Common/Traits/World/ValidateOrder.cs

```diff
@@ -27,6 +28,8 @@ public bool OrderValidation(OrderManager orderManager, World world, int clientId
 
 			var subjectClientId = order.Subject.Owner.ClientIndex;
 			var subjectClient = orderManager.LobbyInfo.ClientWithIndex(subjectClientId);
+			var playerClient = orderManager.LobbyInfo.ClientWithIndex(clientId);
+			var player = world.Players.FirstOrDefault(p => p.ClientIndex == playerClient.Index && p.PlayerReference.Playable);
```

Comment:
Since the uses `FirstOrDefault`, it can return null. We should work out want we want to do this in scenario. Currently we would crash.

Reply:
Is there a possible scenario where this is the case and `subjectClient` is not null? 
I could just add a `if (player == null)` block if you think that is necessary.

Reply:
oh we have a problem. this could happen everywhere where `CanControlUnitsOf` is used and there is no local player

Reply:
should be good with [b66ca53](https://github.com/OpenRA/OpenRA/pull/19144/commits/b66ca53841ed5e102b7ea7a9dfcf4e7dc6be807b).